fixtures/vcr_cassettes/members_by_state.yaml

```diff
@@ -0,0 +1,186 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
```

Comment:
I like the response, does this file get generated?

Reply:
This file was generated after adding `with vcr` before the requests. Then each test references the saved cassette afterward.